src/lib/containers/FacetsPlotNav.tsx

```diff
@@ -0,0 +1,296 @@
+  isResetPossible: boolean,
+}
+
+export type FacetsPlotNavProps = {
+  loadingScreen?: React.FunctionComponent | JSX.Element
+  facetsToPlot?: string[]
+  link?: string
+  linkText?: string
+}
+
+type InternalProps = FacetsPlotNavProps & QueryWrapperChildProps
+/**
+ * Make a chart for each facet
+ *
+ * @class FacetsPlotNav
+ * @extends {React.Component}
+ */
+export default class FacetsPlotNav extends React.Component<
+  InternalProps,
+  FacetsPlotNavState
+  > {
+  constructor(props: InternalProps) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    // the text currently under the cursor
+    this.state = {
+      isShowingMore: false,
+      isResetPossible: false,
+    }
+    this.extractPropsData = this.extractPropsData.bind(this)
+    this.toggleShowMore = this.toggleShowMore.bind(this)
+    this.onReset = this.onReset.bind(this)
+    // this.onHover = this.onHover.bind(this)
+    // this.onUnhover = this.onUnhover.bind(this)
+  }
+
+  public toggleShowMore(event: React.MouseEvent<HTMLAnchorElement>) {
+    event.preventDefault()
+    this.setState({
+      isShowingMore: !this.state.isShowingMore,
+    })
+  }
+
+  public onReset(event: React.MouseEvent<HTMLAnchorElement>) {
+    event.preventDefault()
+    const { executeInitialQueryRequest } = this.props
+    if (executeInitialQueryRequest) {
+      executeInitialQueryRequest()
+    }
+    this.setState({
+      isResetPossible: false
+    })
+  }
+
+  // public onHover = (event: any) => {
+  //   this.updateHover(true, event)
+  // }
+
+  // NOTE: There's an issue where onUnhover() is not called after the first onHover().  
+  // Moving the plotData to state requires some consideration.  The plot must be updated by the table filter selection.
+  // Also, the Plot will not update if it points to the same (shallow compare) data object.  One way to force a Plot update is to
+  // update the layout.datarevision.
+  // public onUnhover = (event: any) => {
+  //   this.updateHover(false, event)
+  // }
+
+  // public updateHover = (isHover:boolean, event: any) => {
+  //   if (event.points && event.points[0] && this.state.plotData) {
+  //     const plotPointData: any = event.points[0]
+  //     const domain = plotPointData.data.domain
+  //     const newMarkerLineWidth: number = isHover ? 1 : 0
+  //     this.state.plotData[(domain.row * CHARTS_PER_ROW) + domain.column].marker.line.width[plotPointData.pointNumber] = newMarkerLineWidth
+  //     this.setState({datarevision: this.state.datarevision + 1})
+  //   }
+  // }
+
+  /**
+   * Handle click event
+   */
+  public handleClick = (event: any) => {
+    if (event.points && event.points[0]) {
+      const plotPointData: any = event.points[0]
+      const facetName = plotPointData.data.name
+      const facetValueClicked = plotPointData.data.facetEnumerationValues[plotPointData.pointNumber]
+      // update the facet and selected index
+      const chartSelectionIndex = plotPointData.pointNumber
```

Comment:
remove chartSelectionIndex update (does not apply, because we are not linking the bar chart to chiclets in this case).